programmers/난이도별/level01.문자열_내_p와_y의_개수/yoojinhee03.java

```diff
@@ -0,0 +1,24 @@
+package programmers;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class yoojinhee03 {
+
+	public static void main(String[] args) {
+		int[] arr= {1,2,3,4,5};
+		System.out.println(solution("pPyyY"));
+		
+	}
+	
+	static boolean solution(String s) {
+
+        s=s.toUpperCase();
+        int cnt=0;
+        String arr[]=s.split("");
+        for(String str:arr) 
+        	cnt= str.equals("P")?cnt+1:str.equals("Y")?cnt-1:cnt+0;  		   

```

Comment:
```suggestion
            cnt = str.equals("P") ? cnt + 1 :
                  str.equals("Y") ? cnt - 1 : cnt;
```
이렇게 표현하면 조금 더 읽기 수월해집니다!